packages/bundler-plugin-core/src/index.ts

```diff
@@ -15,6 +15,7 @@ import {
   createRollupAsset,
   getCompressedSize,
   handleErrors,
+  type NormalizedOptions,
```

Comment:
small edits to `bundler-plugin-core` to export this type used in the `standalone-analyzer` operations